src/applications/personalization/profile/routes.js

```diff
@@ -1,7 +1,7 @@
 import AccountSecurity from './components/account-security/AccountSecurity';
 import PersonalInformation from './components/personal-information/PersonalInformation';
 import MilitaryInformation from './components/military-information/MilitaryInformation';
-import DirectDeposit from './components/direct-deposit/DirectDeposit';
+import DirectDeposit from './components/direct-deposit/DirectDepositWrapper';
```

Comment:
Can you name this `DirectDepositWrapper?`. A bit easier to follow that way

Reply:
done!